Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -575,6 +575,9 @@ massDownload=Massendownload
 massImport=Massenimport
 masterpieceProperties=Werkst\u00FCckeigenschaft
 mediaUploaded={0} wurde erfolgreich hochgeladen
+mediaWillBeAssigned=ausgew\u00E4hlte Medien werden zugewiesen.
```

Comment:
It seems you are not using these new messages keys anywhere. Are they really required?